modules/ROOT/pages/propagate-metadata-studio.adoc

```diff
@@ -0,0 +1,161 @@
+= Propagate Metadata to Other Flows
+
+Studio enables you to propagate your flow's metadata to your referenced flows within your application.
+
+Depending on the metadata configured in your referenced flow, you might need to configure the specific metadata to propagate.
+
```

Comment:
Yeah...in this sentence and the one before it, I wonder if the audience might get confused between "your flow" and "your referenced flow"? Maybe just look again at these sentences?

Reply:
Looking below, I see you use "target": maybe "source flow" and "target flow"?
"Studio enables you to propagate metadata in your source flow to other, target flows referenced within your application. Depending on metadata already in the target flow, you might need to configure the specific metadata to propagate from the source flow."?